keyboards/mt980/mt980.c

```diff
@@ -20,6 +20,18 @@ void led_set_keymap(uint8_t usb_led) {}
 __attribute__ ((weak))
 void action_function_keymap(keyrecord_t *record, uint8_t id, uint8_t opt) {}
 
+void matrix_init_user(void) {
+  matrix_init_keymap();
+}
+
+void matrix_scan_user(void) {
+  matrix_scan_keymap();
+}
+
+void action_function_user(keyrecord_t *record, uint8_t id, uint8_t opt) {
```

Comment:
This doesn't actually exist. 

Also, functions are depreciated, and should be removed.

Reply:
OK removed it